metadata.yml

```diff
@@ -11,10 +11,12 @@ mutating: false
 contextAware: false
 executionMode: kubewarden-wapc
 annotations:
-  io.kubewarden.policy.title: psp-allowed-proc-mount-types
+  io.artifacthub.displayName: Allowed proc mount types psp
```

Comment:
for `displayName`, let's match exactly "Allowed Proc Mount Types PSP", so we don't change the display name on artifacthub. This is the title that ends up in artifacthub.io, and later gets show in
https://artifacthub.io/packages/kubewarden/allowed-proc-mount-types-psp/allowed-proc-mount-types-psp